src/main/java/com/cloudogu/scm/review/comment/service/CommentService.java

```diff
@@ -176,7 +193,9 @@ public void modifyReply(String namespace, String name, String pullRequestId, Str
         Reply clone = reply.clone();
         reply.setComment(changedReply.getComment());
         reply.addTransition(new ExecutedTransition<>(keyGenerator.createKey(), CHANGE_TEXT, System.currentTimeMillis(), getCurrentUserId()));
+        reply.setMentionUserIds(mentionMapper.extractMentionsFromComment(changedReply.getComment()));
```

Comment:
Maybe this could be merged with the postMentionEventIf... method into a `handleMentions` method or something alike